services/director-v2/src/simcore_service_director_v2/modules/dask_clients_pool.py

```diff
@@ -23,7 +23,7 @@
 @dataclass
 class DaskClientsPool:
     app: FastAPI
-    settings: DaskSchedulerSettings
+    settings: DaskComputationalBackendSettings
```

Comment:
same here as in DaskClient

Reply:
not going to do this in this PR. maybe at a later time point, but now this takes hours for no big gain. I might do it in a next iteration.